### PR TITLE
Feature/support json schema

### PR DESCRIPTION
**Pull request by runerys**

### CODE REVIEW

**Review by OysteinThuen — APPROVED** [review]

**Comment by itaykat** [comment]
Yo! What about protobuf? So easy to add, please. Thanks you.

File: schemaregistry/resource_schema.go [hunk 1]
```diff
@@ -91,10 +98,15 @@ func schemaCreate(ctx context.Context, d *schema.ResourceData, meta interface{})
 	subject := d.Get("subject").(string)
 	schemaString := d.Get("schema").(string)
 	references := ToRegistryReferences(d.Get("reference").([]interface{}))
+	schemaType := srclient.Avro
+
+	if d.Get("schema_type").(string) == "json" {
+		schemaType = srclient.Json
+	}
```
Comment (itaykat) [inline schemaregistry/resource_schema.go:106]: if d.Get("schema_type").(string) == "protobuf" {
		schemaType = srclient.Protobuf
	}

**Review by itaykat — COMMENTED** [review]
added some comments for adding protobuf support

**Comment by itaykat** [comment]
Any plans for protobuf support?

**Comment by runerys** [comment]
> Any plans for protobuf support?

Just released a version now where it is allowed. Note that we've just made the minor change that allows it - but we haven't tested that the schema registry allows it.

**Pull request merged**